charts/core/README.md

```diff
@@ -58,66 +84,92 @@ Chart container component.
 
 #### Props
 
-| Name           | Description                                             | Type       | Default |
-| -------------- | ------------------------------------------------------- | ---------- | ------- |
-| `onChartReady` | Callback to be called when chart is finished rendering. | () => void |         |
+| Name           | Description                                             | Type         | Default |
+| -------------- | ------------------------------------------------------- | ------------ | ------- |
+| `onChartReady` | Callback to be called when chart is finished rendering. | `() => void` |         |
```

Comment:
Made some drive-by style changes in the README, mostly to use backticks on some code that wasn't previously formatted